### PR TITLE
added regression tutoring folder

### PR DESCRIPTION
**Pull request by Furentsu**

### CODE REVIEW

File: L37_tutoring/.DS_Store
Comment (lescai) [inline L37_tutoring/.DS_Store:1]: this should be deleted

File: L37_tutoring/regression/.DS_Store
Comment (lescai) [inline L37_tutoring/regression/.DS_Store:1]: this should be deleted

**Review by lescai — CHANGES_REQUESTED** [review]
see Mac files to be removed first

**Review by lescai — APPROVED** [review]

File: L37_tutoring/regression/.DS_Store
Comment (Furentsu) [reply L37_tutoring/regression/.DS_Store:1]: Done

**Pull request merged**